utils/sort_auto_mappings.py

```diff
@@ -46,7 +45,7 @@
 _re_identifier = re.compile(r'\s*\(\s*"(\S[^"]+)"')
 
 
-def sort_auto_mapping(fname: str, overwrite: bool = False) -> Optional[bool]:
+def sort_auto_mapping(fname: str, overwrite: bool = False) -> bool:
```

Comment:
I mean the following, yes, the return signature should be changed too (good catch!)

```
    Returns:
        `Optional[bool]`: Returns `None` if `overwrite=True`. Otherwise returns `True` if the file has an auto-mapping
improperly sorted, `False` if the file is okay.
```

Reply:
Actually put back the optional and fixed the return so it's correct.

Reply:
Good for me!